portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,94 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+    background: rgba(0, 0, 0, 0.9);
+    margin: 0;
+    color: #fff;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* Full screen landing page */
```

Comment:
I love that you have used comments to sparate css styles for different parts of the web page.